src/lib/components/auth-cards/LaunchCard.svelte

```diff
@@ -89,15 +95,15 @@
         textColor="#000000"
         background="transparent"
         borderColor="#000000"
-        custom="card-button"
+        custom = "card-button"
+        btnID = {welcomeCard ? "signin" : "create"}
+        on:click={() => {
+          welcomeCard ? handleTrigger("signin") : handleTrigger("create");
+          console.log("THIS IS A HIT ON LAUNCH CARD")
```

Comment:
Is this just for debugging?

Reply:
Yes! I realized after that I forgot to delete it.